src-version/14.1.1/com/intellij/plugins/haxe/editor/HaxeReferenceCopyPasteProcessor.java

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
@giabao, Did you happen to test this on 14.0?  I think that the fix should apply there as well, since the change was brought into the 139 code line.